tests/unit/libbpf_test.cpp

```diff
@@ -3233,14 +3233,68 @@ TEST_CASE("bind_tail_call_max_exceed", "[libbpf]")
     usersim_trace_logging_set_enabled(false, 0, 0);
 }
 
-TEST_CASE("libbpf map batch", "[libbpf]")
+void
+_test_batch_iteration_maps(fd_t& map_fd, uint32_t batch_size, bpf_map_batch_opts* opts, bool concurrent_delete)
+{
+    // Retrieve in batches.
+    uint32_t* next_key_ptr = nullptr;
+    uint32_t next_key = 0;
+    std::vector<uint32_t> returned_keys(0);
+    std::vector<uint64_t> returned_values(0);
+    uint32_t requested_batch_size = batch_size / 10 - 2;
```

Comment:
"10" and "2" seem to be magic numbers.  Where did they come from?

Reply:
Using small batch size...and also including the last batch with some remaining items to select.  

In this example, total entries is 20,000. 
- Each requested_batch_size is 1998. 
- The final batch retrieval is 20 entries. 
- Total iteration is 11.

Reply:
https://github.com/microsoft/ebpf-for-windows/blob/main/docs/DevelopmentGuide.md says:
> **DON'T** use hard-coded magic numbers for things that have to be consistent between different files. Instead use a #define or an enum or const value, as appropriate.

Reply:
Changed to const.